docs/api/domains.md

```diff
@@ -135,5 +135,25 @@ const Planets = {
   }
 }
 
-repo.push(Actions.add, { name: 'earth' }) // this will add Earth
+repo.push(addPlanet, { name: 'earth' }) // this will add Earth
+```
+
+Registration is given the command associated with an action, and the
```

Comment:
Haha... Yeah. I know... I'm not sure where to go from here. Just calling this `action` instead of `command` works really well for me. It would be better to rename the `Action` class, which doesn't have a public API.